libs/pubsub/query/query.go

```diff
@@ -232,105 +235,119 @@ func (q *Query) Matches(tags map[string]string) bool {
 	return true
 }
 
-// match returns true if the given triplet (tag, operator, operand) matches any tag.
+// match returns true if the given triplet (tag, operator, operand) matches any
+// value in an event for that key.
 //
-// First, it looks up the tag in tags and if it finds one, tries to compare the
-// value from it to the operand using the operator.
+// First, it looks up the key in the events and if it finds one, tries to compare
+// all the values from it to the operand using the operator.
 //
 // "tx.gas", "=", "7", { "tx.gas": 7, "tx.ID": "4AE393495334" }
```

Comment:
The final argument on this line should be updated for the new format

Reply:
Updated.